onnxscript/rewriter/onnxruntime/fused_matmul_rule_sets.py

```diff
@@ -167,13 +167,13 @@ def fused_matmul_rule_sets() -> orp.RewriteRuleSet:
     """
     return orp.RewriteRuleSet(
         [
-            orp.make_rewrite_rule_from_class(FusedMatMulDiv1, True),
-            orp.make_rewrite_rule_from_class(FusedMatMulDiv2, True),
-            orp.make_rewrite_rule_from_class(FusedMatMulTranspose, True),
-            orp.make_rewrite_rule_from_class(MatMulTranspose, True),
-            orp.make_rewrite_rule_from_class(TransposeMatMul1, True),
-            orp.make_rewrite_rule_from_class(TransposeFusedMatMul1, True),
-            orp.make_rewrite_rule_from_class(TransposeMatMul2, True),
-            orp.make_rewrite_rule_from_class(TransposeFusedMatMul2, True),
+            orp.make_rewrite_rule_from_class(FusedMatMulDiv1, generic=True),
```

Comment:
Trying to understand why you have this `generic=True`? Is it for debugging?

Reply:
No it is using the generic pattern matcher (multi output pattern)

Reply:
If the pattern has multiple outputs, the implementation should automatically choose the generic matcher ... user doesn't have to specify it.